src/MATLAB.jl

```diff
@@ -80,7 +80,10 @@ function __init__()
     if libmx_size > 0 # non-zero size library path
 
     # load libraries
-
+    # workaround for https://github.com/JuliaInterop/MATLAB.jl/issues/200
+    if Sys.iswindows()
+        ENV["PATH"] = string(matlab_libpath, ";", ENV["PATH"])
+    end
```

Comment:
Can you run all of this under `withenv`?  Then you could wrap the code below, which uses Libdl, in a function that is called in `withenv`.

The code would look like this:
```julia
function load_matlab_libraries()
	# do the loading here
end

...

# in init
if Sys.iswindows()
withenv(load_matlab_libraries, "PATH" => string(matlab_libpath, ";", ENV["PATH"]))
else
load_matlab_libraries()
```